### ISSUE TITLE
GVRSphereSceneObject, cube, cone and cylinder accepts null for GVRContext parameter

### ISSUE THREAD

**Issue by elidelsonc** [Samsung/GearVRf#107]
GVRSphereSceneObject::GVRSphereSceneObject - accepts null for GVRContext parameter

````
GVRSphereSceneObject sphereSceneObject2 = new GVRSphereSceneObject(null);
```
[GVRSphereSceneObject(GVRContext gvrContext)
Constructs a sphere scene object with a radius of 1 and 18 stacks, and 36 slices.]

(http://docs.gearvrf.org/org/gearvrf/scene_objects/GVRSphereSceneObject.html)

**Comment by JonShemitz** [comment]
> new GVRSphereSceneObject(null);

True, it doesn't test for `null` - but neither do any of the other constructors. Does this constructor do anything differently with `null` than any other GVRF constructor does?

In general, I think getting a `NullPointerException` with the `GVRSphereSceneObject` constructor on the stack ought to be enough of a clue as to what you did wrong. We've added runtime parameter checks in cases where bad parameters could crash the app with a `SIGSEGV` or a `SIGBUS` and no stack trace, but it doesn't seem like that's the issue, here.

In other words, @elidelsonc , this sounds a lot like what Github calls a "wontfix".
